docs/Compiling a configuration.md

```diff
@@ -49,4 +49,10 @@ The script will ask you for the path to your disassembler (IDA), and will sugges
 
 Storing the config file
 -----------------------
-In the end, a new ```*.json``` file will be generated (using the library name + version), and it should be stored together with the rest of the configuration files in the ```configs``` directory.
\ No newline at end of file
+In the end, a new ```*.json``` file will be generated (using the library name + version), and it should be stored together with the rest of the configuration files in the ```configs``` directory.
+
+Uploading the config file back to the community :)
+--------------------------------------------------
+So, you created a new config file and it works on your setup, Good Job :)
+
+Please consider submitting it to the community collection of configurations and fingerprints. Feel free to submit it as a pull request for now, and in the future we might update it to a separate repository.
```

Comment:
Maybe add a link to the right folder on Karta's source? e.g "submitting it to the [community collection](#) of configurations and fingerprints. ..."

Reply:
I considered adding one, but it is listed right above it already under section `storing the config file`. I did told them to submit it in a pull request so we would get it too.

Reply:
right, great! :)
Hopefully it will bring some additions